### ISSUE TITLE
More simple examples in documentation

### ISSUE THREAD

**Issue by brian-rose** [brian-rose/climlab#49]
As of release v0.5.0, the docs are in much better shape in terms of coverage.

However the docs focus too much on climlab internals and not enough on the user interface. We should have simple working examples in the docs for every single process.

I mean more pages that look like this:
http://climlab.readthedocs.io/en/latest/api/climlab.radiation.rrtm.html
and fewer pages that look like this:
http://climlab.readthedocs.io/en/latest/api/climlab.surface.turbulent.html

Basically it would be great to present simple working examples before technical details wherever possible.

**Comment by brian-rose** [comment]
Doing some cleanup here...

Good page:
- https://climlab.readthedocs.io/en/latest/api/climlab.radiation.RRTMG.html

Bad page:
- https://climlab.readthedocs.io/en/latest/api/climlab.radiation.water_vapor.html

Improvements have occurred, but slowly... In particular this page is much better than it used to be:
- https://climlab.readthedocs.io/en/latest/api/climlab.surface.turbulent.html
